src/symbols.ts

```diff
@@ -368,4 +368,10 @@ export const SYMBOLS = {
     parameters: ["pointer", "i32"],
     result: "pointer",
   },
+
+  PyTuple_Pack: {
+    // TODO: This function should have variable length arguments
+    parameters: ["i32", "pointer", "pointer"],
+    result: "pointer",
+  },
```

Comment:
Didn't really know what to do here.

https://docs.python.org/3/c-api/tuple.html#c.PyTuple_Pack

Reply:
Make this `PyTuple_Pack: { type: "pointer" }` and `symbols.PyTuple_Pack` will become an `Deno.UnsafePointer`, which you can further use with combination with `Deno.UnsafeFnPointer` to create dynamic definitions for the symbol based on the number of arguments you got.

Reply:
Great! Thanks for the info! I modified it and it now works for any size.